### PR TITLE
[frigate] update frigate version to 0.9.4

### PR DESCRIPTION
**Pull request by PixelJonas**
**Description of the change**

This PR bumps frigate version to 0.9.4 and the chart-version to 7.0.0 as frigate introduced breaking changes to the config file. README.md is adjusted accordingly

**Benefits**

I just started installing frigate and there was a big mismatch between frigate documentation of configuration and the one needed for the version used in this chart. The new frigate version also includes better error messages for missing configuration, which should make the upgrade easy.

**Additional information**

Most of my configuration errors were based on custom camera configuration.

*v0.8.x*
```
...
camera:
  width: 640
  height: 360
  fps: 4
...
```

*v0.9.x*
```
...
camera:
  detect:
    width: 640
    height: 360
    fps: 4
...
```

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Variables are documented in the README.md (this can be done with using our helm-docs wrapper `./hack/gen-helm-docs.sh stable <chart>`)

### CODE REVIEW

**Comment by PixelJonas** [comment]
mhhh chart installs fails because there is no camera configured per default. I don't know if our `values.yaml` should configure any default camera as the camera has to have a unique key which may introduce faulty configuration to users.


```
[2021-12-12 10:20:34] frigate.app                    INFO    : Starting Frigate (0.9.4-26ae608)
*************************************************************
*************************************************************
***    Your config file is not valid!                     ***
***    Please check the docs at                           ***
***    https://docs.frigate.video/configuration/index     ***
*************************************************************
*************************************************************
***    Config Validation Errors                           ***
*************************************************************
1 validation error for FrigateConfig
cameras
  field required (type=value_error.missing)
*************************************************************
***    End Config Validation Errors                       ***
*************************************************************
```

**Comment by onedr0p** [comment]
Before doing the above you can create a `ci/ct-values.yaml` with values that only get used in the CI.

**Comment by PixelJonas** [comment]
let me check if I can find a configuration for a dummy-camera and update the PR

**Comment by PixelJonas** [comment]
can you elaborate on the `incomplete-docs` tag? Did I miss something? AFAIK I updated the `version`, added the `README_CHANGELOG.md.gotmpl` entry 🤔

**Comment by onedr0p** [comment]
Happy trigger finger on me I guess lol. I would suggest not formatting the yaml files. I'm seeing some changes there.

File: charts/stable/frigate/Chart.yaml [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-- tensorflow
-- coral
-- ml
-- machine-learning
+  - frigate
+  - nvr
+  - tensorflow
+  - coral
+  - ml
+  - machine-learning
```
Comment (onedr0p) [inline charts/stable/frigate/Chart.yaml:13]: Don't need to indent

File: charts/stable/frigate/Chart.yaml [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/frigate
 icon: https://github.com/blakeblackshear/frigate/blob/master/docs/static/img/frigate.png
 sources:
-- https://github.com/blakeblackshear/frigate
+  - https://github.com/blakeblackshear/frigate
```
Comment (onedr0p) [inline charts/stable/frigate/Chart.yaml:17]: Don't need to indent

File: charts/stable/frigate/Chart.yaml [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
 maintainers:
-- name: billimek
-  email: jeff@billimek.com
+  - name: billimek
+    email: jeff@billimek.com
+  - name: PixelJonas
+    email: 5434875+PixelJonas@users.noreply.github.com
```
Comment (onedr0p) [inline charts/stable/frigate/Chart.yaml:22]: Don't need to indent

File: charts/stable/frigate/Chart.yaml [hunk 4]
```diff
@@ -1,24 +1,26 @@
...
 dependencies:
-- name: common
-  repository: https://library-charts.k8s-at-home.com
-  version: 4.2.0
+  - name: common
+    repository: https://library-charts.k8s-at-home.com
+    version: 4.2.0
```
Comment (onedr0p) [inline charts/stable/frigate/Chart.yaml:26]: Don't need to indent

File: charts/stable/frigate/values.yaml [hunk 5]
```diff
@@ -82,7 +82,8 @@ affinity: {}
 #           - coral-tpu
 
 # -- Configure the resource requests and/or limits for the Pod
-resources: {}
+resources:
+  {}
```
Comment (onedr0p) [inline charts/stable/frigate/values.yaml:86]: Formatting should have the {} on the same like like before

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
Thanks!

**Pull request merged**